main.py

```diff
@@ -212,22 +201,26 @@ async def info(interaction: Interaction, name: app_commands.Choice[str], details
     try:
         stats = server_data.__getattr__(server)
     except AttributeError:
-        interaction.response.send_message('Requested server could not be found')
+        return interaction.response.send_message('Requested server could not be found')
```

Comment:
I added a `return` here to ensure `stats` would not be undefined when called.

Reply:
We're not expecting any return value from this function, so don't return any, plus interaction.response.send_message is async and needs to be awaited:
```suggestion
        await interaction.response.send_message('Requested server could not be found')
        return
```